db/change_cache.go

```diff
@@ -645,12 +649,14 @@ func (c *changeCache) processEntry(change *LogEntry) base.Set {
 	}
 
 	sequence := change.Sequence
+	c.context.DbStats.StatsDatabase().Set(base.StatKeyDcpMaxReceived, base.ExpvarUInt64Val(change.Sequence))
```

Comment:
Need to validate that change.Sequence is higher than the previous value before updating.